### ISSUE TITLE
Parse Error

### ISSUE THREAD

**Issue by jpeyper** [zendesk/maxwell#262]
I'm still running on v0.17.0, but I got this parse error today:

```
2016-02-03T11:12:42,878 ERROR com.zendesk.maxwell.schema.ddl.SchemaChange: Error parsing SQL: '-- Refresh the mysql access changes
FLUSH PRIVILEGES'
```

Has this been fixed by one of the v1.0.0-PRE releases?

**Comment by osheroff** [comment]
no, not yet.  it's similar to another one around SQL comments and the
blacklist.  will get a patch in for the next PRE release

**Comment by jpeyper** [comment]
Cool.

**Comment by osheroff** [comment]
fixed in 1.0.0-RC1
